servicecontrol/transports.md

```diff
@@ -99,6 +99,10 @@ The following ServiceControl connection string options are available:
 * `S3KeyPrefix=<value>` - S3 key prefix [option](/transports/sqs/configuration-options.md#offload-large-messages-to-s3-key-prefix).
 * `DoNotWrapOutgoingMessages=true` - Do not wrap outgoing messages [option](/transports/sqs/configuration-options.md#do-not-wrap-message-payload-in-a-transport-envelope).
 
+> [!NOTE]
+> When using SQS as a transport, for local development purposes it is possible to set up ServiceControl to connect to a LocalStack instance.
```

Comment:
Isn't there somewhere in the SQS transport docs that can link here too?

Reply:
There isn't, as far as I can tell. I added this https://github.com/Particular/docs.particular.net/commit/62c5a2d099896ffc03f500b41b7bff7a05a43a80, but it looks like it's something every transport should list. Isn't that better suited for the "transport at a glance" table?

Reply:
Local development as a new item in each transport at a glance? Could be interesting. ASB could point to that open GitHub issue.

Reply:
> Local development as a new item in each transport at a glance?

That and a row for ServiceControl support. For example, PostgreSQL transport did not have it when released initially.

Reply:
We decided to drop the commit linking to the ServiceControl configuration, it was off-topic for this PR. We'll add a commit to edit the transport and persistence at a glance tables